[FEATURE] Implementing Python code snippets under test for "https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/fluent/data_assets/how_to_organize_batches_in_a_file_based_data_asset" (#7907)
great-expectations/great_expectations: docs/docusaurus/docs/components/connect_to_data/filesystem/_tip_filesystem_data_asset_if_batching_regex_matches_multiple_files.md

For example:

Let's say that you have a filesystem Datasource pointing to a base folder that contains the following files:
- "yellow_tripdata_sample_2019-03.csv"
- "yellow_tripdata_sample_2020-07.csv"
- "yellow_tripdata_sample_2021-02.csv"

If you define a Data Asset using the full file name with no regex groups, such as "yellow_tripdata_sample_2019-03\.csv" your Data Asset will contain only one Batch, which will correspond to that file.

> However, if you define a partial file name with a regex group, such as `r"yellow_tripdata_sample_(?P<year>\d{4})-(?P<month>\d{2}).csv"` your Data Asset can be organized ("partitioned") into Batches according to the two dimensions, defined by the group names, `"year"` and `"month"`.  When you send a Batch Request query featuring this Data Asset in the future, you can use these group names with their respectiv3e values as options to control which Batches will be returned.

Reviewer: ```suggestion
However, if you define a partial file name with a regex group, such as `r"yellow_tripdata_sample_(?P<year>\d{4})-(?P<month>\d{2}).csv"` your Data Asset can be organized ("partitioned") into Batches according to the two dimensions, defined by the group names, `"year"` and `"month"`.  When you send a Batch Request query featuring this Data Asset in the future, you can use these group names with their respective values as options to control which Batches will be returned.
```

> For example, you will be able to return all Batches in the year of 2021, or the one Batch for July of 2020.

Reviewer: ```suggestion
For example, you could return all Batches in the year of 2021, or the one Batch for July of 2020.
```
